src/CompositeRheologies/CompositeRheology.jl

```diff
@@ -81,6 +81,25 @@ compute_εII(v::CompositeRheology{T,N}, τII::_T, args; tol=1e-6, verbose=false)
 compute_εII(v::CompositeRheology{T,N}, τII::Quantity, args; tol=1e-6, verbose=false) where {T,N} = nreduce(vi -> first(compute_εII(vi, τII, args)), v.elements)
 compute_εII(v::AbstractMaterialParamsStruct, τII, args) = compute_εII(v.CompositeRheology[1], τII, args)
 
+# compute strain rate partitioning
+compute_elements_εII(v::CompositeRheology{T,N}, τII, args) where {T, N} = ntuple(i -> first(compute_εII( v.elements[i], τII, args)), Val(N));
+
+"""
+    ε_part = compute_elements_εII(v::NTuple{N1,AbstractMaterialParamsStruct}, τII::NTuple{N2,T}, args, phase::I)
+This returns the individual strainrate components of the `CompositeRheology` defined in `v`
+"""
+function compute_elements_εII(
```

Comment:
This can be just
```julia
function compute_elements_εII(
    v::NTuple{N,AbstractMaterialParamsStruct},
    τII,
    args,
    phase::Int,
) where N